components/HomePage/11EcosystemBuilders/index.jsx

```diff
@@ -68,19 +73,17 @@ const WhyBuildOnAutonolas = () => (
         }, index) => (
           <Col key={imageUrl} lg={12} xs={24}>
             <div className={`column column-${index + 1}`}>
-
               <div className="img-container-custom">
-                <img
+                <Image
                   src={`/images/11EcosystemBuilders/${imageUrl}.png`}
                   alt={`${heading} Icon`}
+                  width={140}
+                  height={140}
```

Comment:
Guess this is ok for now.

Looking a little more into it, it looks like `sizes` is potentially the right way to go
<img width="516" alt="image" src="https://user-images.githubusercontent.com/66292936/218130098-c3ce831e-4217-4d5b-ab4d-0caabacf1de7.png">

Reply:
Yes that's a fair concern, and agree it makes sense to have a helper function. Not sure on the specific implementation, can address that when we come to fix it.